pandas/tests/frame/test_operators.py

```diff
@@ -1161,6 +1161,21 @@ def test_inplace_ops_identity(self):
         assert_frame_equal(df2, expected)
         assert df._data is df2._data
 
+        # no id change
+        arr = np.array([1., 2., 3.])
+        df = DataFrame({'a': arr.copy()})
+        expected = id(df)
+        df += 2.
+        assert id(df) == expected
+        df -= 2.
+        assert id(df) == expected
+        df *= 2.
+        assert id(df) == expected
+        df /= 2.
+        assert id(df) == expected
+        df **= 2.
```

Comment:
add for // as well

Reply:
I added more missing inplace operators (ifloordiv, imod, iand, ior, ixor).  The only ones excluded now are iconcat, ilshift and irshift which I believe are not supported.